interface/forms/newpatient/common.php

```diff
@@ -86,16 +86,35 @@ function refreshIssue(issue, title) {
   s.options[s.options.length] = new Option(title, issue, true, true);
  }
 
+ <?php
+ //Get new_validate option to check if global new validation is on.
+ $new_validate = $GLOBALS['new_validate'] ? 1 : 0;
+ //Gets validation rules from Page Validation list.
+ $collectthis = collectValidationPageRules("/interface/forms/newpatient/new.php");
+ ?>
+ <?php if($new_validate && $collectthis): ?>
```

Comment:
Note that the validations are turned on/off in the Page Validation list via the Active flag. The turning on/off of the validation LBF engine makes sense via a global since a user can can go back to older validation. But, in the Page validation, this is a much different story. There are 3 possibilities for each page:
1. Already complicated validation being done, but wish to add the new validation option (thus will add entry to Page Validation list but turn if off by default) 
2. Simple validation currently being done, and wish to add support for new validation option. In this case there would be an argument to remove the current validation and only use the new validation (thus making it active on the Page Validation list)
3. And the an option in betweent 1 and 2 where will ant to support the new validation and old validation (thus making it inactive on the Page Validation list).
Overall, see no benefit for mixing the global to turn on/off the LBF validation with the Page validation.